system/Autoloader/Autoloader.php

```diff
@@ -210,7 +191,24 @@ public function removeNamespace(string $namespace)
 		return $this;
 	}
 
-	//--------------------------------------------------------------------
+	/**
+	 * Load a class using available class mapping.
+	 *
+	 * @param string $class
+	 *
+	 * @return string|false
+	 */
+	public function loadClassmap(string $class)
+	{
+		$file = $this->classmap[$class] ?? '';
+
+		if ($file !== '')
```

Comment:
While it should never happen, I think `is_string($file)` check need to be added to avoid falsy `$file` for saver